src/index.js

```diff
@@ -46,22 +46,19 @@ async function run(octokit, context, token) {
 		stripHash: stripHash(getInput('strip-hash'))
 	});
 
-	const buildScript = getInput('build-script') || 'build';
+	const buildScript = getInput('build-script', { required: true });
```

Comment:
Just for my own curiosity: what's the purpose behind setting `{ required: true }`? I had assumed the change you made to the yml would cover that already.